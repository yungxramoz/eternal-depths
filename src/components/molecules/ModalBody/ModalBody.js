import React from 'react'
import './ModalBody.css'

const ModalBody = ({ children }) => <div className="modal-body">{children}</div>

export default ModalBody
