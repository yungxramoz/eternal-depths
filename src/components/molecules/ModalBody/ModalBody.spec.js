import React from 'react'
import { render } from '@testing-library/react'
import ModalBody from './ModalBody'

describe('ModalBody', () => {
  it('renders correctly', () => {
    render(<ModalBody />)
    expect(document.body).toMatchSnapshot('ModalBody')
  })
})
