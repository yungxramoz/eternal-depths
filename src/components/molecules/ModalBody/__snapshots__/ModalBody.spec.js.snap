// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBody renders correctly: ModalBody 1`] = `
<body>
  <div>
    <div
      class="modal-body"
    />
  </div>
</body>
`;
